orchestration/src/main/java/com/sap/ai/sdk/orchestration/client/model/FilterConfig.java

```diff
@@ -12,249 +12,17 @@
-
-  /**
-   * Set an unrecognizable property of this {@link FilterConfig} instance. If the map previously
-   * contained a mapping for the key, the old value is replaced by the specified value.
-   *
-   * @param customFieldName The name of the property
-   * @param customFieldValue The value of the property
-   */
-  @JsonIgnore
-  public void setCustomField(@Nonnull String customFieldName, @Nullable Object customFieldValue) {
-    cloudSdkCustomFields.put(customFieldName, customFieldValue);
-  }
-
-  @Override
-  public boolean equals(@Nullable final java.lang.Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    final FilterConfig filterConfig = (FilterConfig) o;
-    return Objects.equals(this.cloudSdkCustomFields, filterConfig.cloudSdkCustomFields)
-        && Objects.equals(this.type, filterConfig.type)
-        && Objects.equals(this.config, filterConfig.config);
-  }
-
-  @Override
-  public int hashCode() {
-    return Objects.hash(type, config, cloudSdkCustomFields);
-  }
-
-  @Override
-  @Nonnull
-  public String toString() {
-    final StringBuilder sb = new StringBuilder();
-    sb.append("class FilterConfig {\n");
-    sb.append("    type: ").append(toIndentedString(type)).append("\n");
-    sb.append("    config: ").append(toIndentedString(config)).append("\n");
-    cloudSdkCustomFields.forEach(
-        (k, v) ->
-            sb.append("    ").append(k).append(": ").append(toIndentedString(v)).append("\n"));
-    sb.append("}");
-    return sb.toString();
-  }
-
-  /**
-   * Convert the given object to string with each line indented by 4 spaces (except the first line).
-   */
-  private String toIndentedString(final java.lang.Object o) {
-    if (o == null) {
-      return "null";
-    }
-    return o.toString().replace("\n", "\n    ");
-  }
-
-  /**
-   * Create a type-safe, fluent-api builder object to construct a new {@link FilterConfig} instance
-   * with all required arguments.
-   */
-  public static Builder create() {
-    return (type) -> new FilterConfig().type(type);
-  }
-
-  /** Builder helper class. */
-  public interface Builder {
-    /**
-     * Set the type of this {@link FilterConfig} instance.
-     *
-     * @param type String represents name of the filter provider
-     * @return The FilterConfig instance.
-     */
-    FilterConfig type(@Nonnull final TypeEnum type);
-  }
-}
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@javax.annotation.Generated(
+    value = "org.openapitools.codegen.languages.JavaClientCodegen",
+    date = "2024-11-08T10:43:10.628772+01:00[Europe/Berlin]",
+    comments = "Generator version: 7.9.0")
+@JsonIgnoreProperties(
+    value = "", // ignore manually set , it will be automatically generated by Jackson during
+    // serialization
+    allowSetters = true // allows the  to be set during deserialization
+    )
```

Comment:
(Minor)

Maybe we can remove this. To not give wrong impression about this class being generated.